### PR TITLE
Clarifications

### PR DESCRIPTION
**Pull request by fbiville**

### CODE REVIEW

File: bolt/bolt-protocol-message-specification-4.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Protocol errors are fatal and should immediately transition the server state to
 
 ## Message Exchange
 
-Messages are exchanged in a request-response pattern between client and server.
+Messages are exchanged in a request-response pattern between the client and server.
```
Comment (robsdedude) [inline bolt/bolt-protocol-message-specification-4.md:84]: Either is proper English. If you prefer the definite form (with article) I'd argue to apply it to both nouns: "the server and the client".

Personally, I prefer the version without articles.

File: bolt/bolt-protocol-message-specification-4.md [hunk 2]
```diff
@@ -500,11 +500,11 @@ extra::Dictionary(
...
-  - The `tx_metadata` is a dictionary that can contain some metadata information, mainly used for logging.
-  - The `mode` specifies what kind of server the `RUN` message is targeting. For write access use `"w"` and for read access use `"r"`. Defaults to write access if no mode is sent.
-  - The `db` specifies the database name for multi-database to select where the transaction takes place. If no `db` is sent or empty string it implies that it is the default database.
+  - `bookmarks`: it is a list of strings containing some kind of bookmark identification e.g. ["neo4j-bookmark-transaction:1", "neo4j-bookmark-transaction:2"]
+  - `tx_timeout`: it is an integer in that specifies a transaction timeout in ms.
+  - `tx_metadata`: it is a dictionary that can contain some metadata information, mainly used for logging.
+  - `mode`: it specifies what kind of server the `RUN` message is targeting. For write access use `"w"` and for read access use `"r"`. Defaults to write access if no mode is sent.
+  - `db`: it specifies the database name for multi-database to select where the transaction takes place. If no `db` is sent or empty string it implies that it is the default database.
```
Comment (robsdedude) [inline bolt/bolt-protocol-message-specification-4.md]: I think removing "it is" or "it" at the beginning of each item completely is even more concise.

File: bolt/bolt-protocol-message-specification-4.md [hunk 3]
```diff
@@ -500,11 +500,11 @@ extra::Dictionary(
 
   For **Auto-commit Transaction** (`RUN`) the `extra` field carries:
 
-  - The `bookmarks` is a list of strings containg some kind of bookmark identification e.g ["neo4j-bookmark-transaction:1", "neo4j-bookmark-transaction:2"]
-  - The `tx_timeout` is an integer in that specifies a transaction timeout in ms.
-  - The `tx_metadata` is a dictionary that can contain some metadata information, mainly used for logging.
-  - The `mode` specifies what kind of server the `RUN` message is targeting. For write access use `"w"` and for read access use `"r"`. Defaults to write access if no mode is sent.
-  - The `db` specifies the database name for multi-database to select where the transaction takes place. If no `db` is sent or empty string it implies that it is the default database.
+  - `bookmarks`: it is a list of strings containing some kind of bookmark identification e.g. ["neo4j-bookmark-transaction:1", "neo4j-bookmark-transaction:2"]
+  - `tx_timeout`: it is an integer in that specifies a transaction timeout in ms.
```
Comment (robsdedude) [inline bolt/bolt-protocol-message-specification-4.md]: integer ~~in~~ that

File: packstream/packstream-specification-1.md [hunk 4]
```diff
@@ -182,7 +182,7 @@ Specifically, values between `00` and `7F` inclusive can be directly translated
 
 Similarly, values between `F0` and `FF` inclusive can do the same for negative numbers between -16 and -1.
 
-Note that while it is possible to encode small numbers in wider formats, it is generally recommended to use the most compact representation possible.
```
Comment (robsdedude) [inline packstream/packstream-specification-1.md:185]: The to-infinitive seemed right here to me.

File: packstream/packstream-specification-1.md [hunk 5]
```diff
@@ -517,13 +518,13 @@ Example:
 ## Structure
 
 
-A structure is a composite value, comprised of fields and a unique type code.
+A structure is a composite value, comprised of a unique type code, and a sequence of fields.
```
Comment (robsdedude) [inline packstream/packstream-specification-1.md]: I'm bad with punctuation. Does the comma before "and" belong there?

**Review by robsdedude — CHANGES_REQUESTED** [review]
Good additions overall.

File: bolt/bolt-protocol-message-specification-4.md (hunk 1 above)
Comment (fbiville) [reply bolt/bolt-protocol-message-specification-4.md:84]: > Both is proper English.

The spellchecker of IntelliJ does not seem to agree :)
I can revert that change though, no strong opinion here.

File: packstream/packstream-specification-1.md (hunk 5 above)
Comment (fbiville) [reply packstream/packstream-specification-1.md]: Screenshot 2021-03-23 at 17 23 13

File: packstream/packstream-specification-1.md (hunk 4 above)
Comment (fbiville) [reply packstream/packstream-specification-1.md:185]: Screenshot 2021-03-23 at 17 24 29

File: packstream/packstream-specification-1.md (hunk 5 above)
Comment (johannessen) [reply packstream/packstream-specification-1.md]: So then a “sequence of fields” is not part of the “composite value”? That seems dubious…

Comment (fbiville) [reply packstream/packstream-specification-1.md]: oh, wait, those are not independent clauses indeed :)

**Pull request closed without merging**